docs/advanced/retry-operations.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Retrying Operations
+order: 4
+---
+
+# Retrying Operations
+
+The `retryExchange` lets us retry specific operation, by default it will
+retry only `network errors` but we can specify additional options to add
+functionality.
+
+## Installation and Setup
+
+First install `@urql/exchange-retry` alongside `urql`:
+
+```sh
+yarn add @urql/exchange-retry
+# or
+npm install --save @urql/exchange-retry
+```
+
+You'll then need to add the `retryExchange`, exposed by this package, to your `urql` Client:
+
+```js
+import { createClient, dedupExchange, cacheExchange, fetchExchange } from 'urql';
+import { retryExchange } from '@urql/exchange-retry';
+
+// None of these options have to be added, these are the default values.
+const options = {
+  initialDelayMs: 1000,
+  maxDelayMs: 15000,
+  randomDelay: true,
+  maxNumberAttempts: 2,
+  retryIf: err => err && err.networkError,
+};
+
+const client = createClient({
+  url: 'http://localhost:1234/graphql',
+  exchanges: [
+    dedupExchange,
+    cacheExchange,
+    fetchExchange,
+    retryExchange(options), // Use the retryExchange factory to add a new exchange
+  ],
+});
+```
+
+We want to place the `retryExchange` after the `fetchExchange` so that retries are only performed _after_ the operation has passed through the cache and has attempted to fetch.
+
+## The Options
```

Comment:
Do we want to add an API page for this package instead? It seems more appropriate to demo some use-cases via the options instead of listing all of them which would typically be more fitting for the API page with a nice table 🤔

Reply:
Do you like how it was before? https://github.com/FormidableLabs/urql/pull/599/commits/b944d3f32b487484a92dd7c47c551afcdb561764

Reply:
I'm happy to add these into the API section, and revert it to Jovi's less-API focussed description if that works?

Reply:
Oh, yea, if we had a shorter section before, let's split them up again? so restore some of the old content and move the new one to an API doc? :)